### PR TITLE
More examples

### PR DESCRIPTION
**Pull request by codewithgun**

### CODE REVIEW

File: ts-client/src/examples/createCustomizableDlmmWithPermissionlessVault.ts [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+import {
+  Clock,
+  ClockLayout,
+  createDummyMint,
+  loadKeypairFromFile,
+} from "./utils";
+import { NATIVE_MINT } from "@solana/spl-token";
+import { AlphaVault, PoolType } from "../alpha-vault";
+
+async function createCustomizableDlmmWithPermissionlessVault(
```
Comment (andrewsource147) [inline ts-client/src/examples/createCustomizableDlmmWithPermissionlessVault.ts:25]: better to make function take parameters as user input, and in function in line 123, user can set necessary parameters

File: ts-client/src/examples/createCustomizableDynamicPoolWithPermissionlessVault.ts [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+  loadKeypairFromFile,
+} from "./utils";
+import { NATIVE_MINT } from "@solana/spl-token";
+import { AlphaVault, DYNAMIC_AMM_PROGRAM_ID, PoolType } from "../alpha-vault";
+import AmmImpl from "@mercurial-finance/dynamic-amm-sdk";
+import { ActivationType } from "@mercurial-finance/dynamic-amm-sdk/dist/cjs/src/amm/types";
+import { deriveCustomizablePermissionlessConstantProductPoolAddress } from "@mercurial-finance/dynamic-amm-sdk/dist/cjs/src/amm/utils";
+import BN from "bn.js";
+
+async function createCustomizableDynamicPoolWithPermissionlessVault(
```
Comment (andrewsource147) [inline ts-client/src/examples/createCustomizableDynamicPoolWithPermissionlessVault.ts:24]: ditto^

File: ts-client/src/examples/permissioned/authorityFund/createCustomizableDlmmWithPermissionlessVault.ts [hunk 3]
```diff
@@ -0,0 +1,127 @@
...
+import {
+  Clock,
+  ClockLayout,
+  createDummyMint,
+  loadKeypairFromFile,
+} from "./../../utils";
+import { NATIVE_MINT } from "@solana/spl-token";
+import { AlphaVault, PoolType } from "../../../alpha-vault";
+
+async function createCustomizableDlmmWithPermissionlessVault(
```
Comment (andrewsource147) [inline ts-client/src/examples/permissioned/authorityFund/createCustomizableDlmmWithPermissionlessVault.ts:25]: ditto^

File: ts-client/src/examples/permissioned/authorityFund/createCustomizableDynamicPoolWithPermissionedVault.ts [hunk 4]
```diff
@@ -0,0 +1,133 @@
...
+  AlphaVault,
+  DYNAMIC_AMM_PROGRAM_ID,
+  PoolType,
+} from "../../../alpha-vault";
+import AmmImpl from "@mercurial-finance/dynamic-amm-sdk";
+import { ActivationType } from "@mercurial-finance/dynamic-amm-sdk/dist/cjs/src/amm/types";
+import { deriveCustomizablePermissionlessConstantProductPoolAddress } from "@mercurial-finance/dynamic-amm-sdk/dist/cjs/src/amm/utils";
+import BN from "bn.js";
+
+async function createCustomizableDynamicPoolWithPermissionedVault(
```
Comment (andrewsource147) [inline ts-client/src/examples/permissioned/authorityFund/createCustomizableDynamicPoolWithPermissionedVault.ts:28]: ditto^

File: ts-client/src/examples/permissioned/merkleRoot/createCustomizableDynamicPoolWithPermissionedVault.ts [hunk 5]
```diff
@@ -0,0 +1,133 @@
...
+  AlphaVault,
+  DYNAMIC_AMM_PROGRAM_ID,
+  PoolType,
+} from "../../../alpha-vault";
+import AmmImpl from "@mercurial-finance/dynamic-amm-sdk";
+import { ActivationType } from "@mercurial-finance/dynamic-amm-sdk/dist/cjs/src/amm/types";
+import { deriveCustomizablePermissionlessConstantProductPoolAddress } from "@mercurial-finance/dynamic-amm-sdk/dist/cjs/src/amm/utils";
+import BN from "bn.js";
+
+async function createCustomizableDynamicPoolWithPermissionedVault(
```
Comment (andrewsource147) [inline ts-client/src/examples/permissioned/merkleRoot/createCustomizableDynamicPoolWithPermissionedVault.ts:28]: ditto^

File: CHANGELOG.md [hunk 6]
```diff
@@ -19,6 +19,18 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
...
+
+### Fixed
+
+- Fix `createStakeEscrowByAuthority` missing parameter `vaultAuthority`
+
+### Added
+
+- Added function `createMultipleStakeEscrowByAuthorityInstructions` to allow vault creator to create multiple stake escrow accounts in single transaction.
+- Added function `createCustomizableFcfsPermissionlessVault`, `createCustomizableFcfsPermissionedVaultWithMerkleProof`, `createCustomizableFcfsPermissionedVaultWithAuthorityFund` to create FCFS vault with customizable parameters
+- Added function `createCustomizableProrataPermissionlessVault`, `createCustomizableProrataPermissionedVaultWithMerkleProof`, `createCustomizableProrataPermissionedVaultWithAuthorityFund` to create prorata vault with customizable parameters
```
Comment (andrewsource147) [inline CHANGELOG.md:32]: ditto^

File: CHANGELOG.md [hunk 7]
```diff
@@ -19,6 +19,18 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
...
+## @meteora-ag/alpha-vault [1.1.6] [PR #6](https://github.com/MeteoraAg/alpha-vault-sdk/pull/6)
+
+### Fixed
+
+- Fix `createStakeEscrowByAuthority` missing parameter `vaultAuthority`
+
+### Added
+
+- Added function `createMultipleStakeEscrowByAuthorityInstructions` to allow vault creator to create multiple stake escrow accounts in single transaction.
+- Added function `createCustomizableFcfsPermissionlessVault`, `createCustomizableFcfsPermissionedVaultWithMerkleProof`, `createCustomizableFcfsPermissionedVaultWithAuthorityFund` to create FCFS vault with customizable parameters
```
[the remark this answers is not in the document]
Comment (codewithgun) [reply CHANGELOG.md]: Fixed

File: CHANGELOG.md (hunk 6 above)
Comment (codewithgun) [reply CHANGELOG.md]: Fixed

File: ts-client/src/examples/permissioned/merkleRoot/createCustomizableDynamicPoolWithPermissionedVault.ts (hunk 5 above)
Comment (codewithgun) [reply ts-client/src/examples/permissioned/merkleRoot/createCustomizableDynamicPoolWithPermissionedVault.ts:29]: Skipped due to it's example entrypoint

File: ts-client/src/examples/permissioned/merkleRoot/createCustomizableDlmmWithPermissionedVault.ts [hunk 8]
```diff
@@ -0,0 +1,127 @@
...
+import {
+  Clock,
+  ClockLayout,
+  createDummyMint,
+  loadKeypairFromFile,
+} from "./../../utils";
+import { NATIVE_MINT } from "@solana/spl-token";
+import { AlphaVault, PoolType } from "../../../alpha-vault";
+
+async function createCustomizableDlmmWithPermissionedVault(
```
[the remark this answers is not in the document]
Comment (codewithgun) [reply ts-client/src/examples/permissioned/merkleRoot/createCustomizableDlmmWithPermissionedVault.ts:29]: Skipped due to it's example entrypoint

File: ts-client/src/examples/permissioned/authorityFund/createCustomizableDlmmWithPermissionlessVault.ts (hunk 3 above)
Comment (codewithgun) [reply ts-client/src/examples/permissioned/authorityFund/createCustomizableDlmmWithPermissionlessVault.ts]: Skipped due to it's example entrypoint

File: ts-client/src/examples/createCustomizableDynamicPoolWithPermissionlessVault.ts (hunk 2 above)
Comment (codewithgun) [reply ts-client/src/examples/createCustomizableDynamicPoolWithPermissionlessVault.ts:29]: Skipped due to it's example entrypoint

File: ts-client/src/examples/createCustomizableDlmmWithPermissionlessVault.ts (hunk 1 above)
Comment (codewithgun) [reply ts-client/src/examples/createCustomizableDlmmWithPermissionlessVault.ts:25]: Skipped due to it's example entrypoint

File: ts-client/src/alpha-vault/index.ts [hunk 9]
```diff
@@ -86,6 +89,163 @@ export class AlphaVault {
...
+   * @param {PublicKey} owner - The owner of the vault.
+   * @param {WhitelistMode} whitelistMode - The whitelist mode for the vault. 0 = Permissionless, 1 = PermissionWithMerkleProof, 2 = PermissionWithAuthority
+   * @param {Opt} [opt] - Optional configuration options.
+   * @return {Promise<Transaction>} The transaction for creating the vault.
+   */
+  public static async createCustomizableProrataVault(
+    connection: Connection,
+    vaultParam: CustomizableProrataVaultParams,
+    owner: PublicKey,
+    whitelistMode: WhitelistMode,
```
Comment (andrewsource147) [inline ts-client/src/alpha-vault/index.ts:186]: Why whitelistMode is not in vaultParams, but in a new parameter?

Comment (codewithgun) [reply ts-client/src/alpha-vault/index.ts]: Fixed

**Review by andrewsource147 — APPROVED** [review]

**Pull request merged**